site/guide/work-with-content-blocks.qmd

```diff
@@ -50,6 +50,9 @@ Test-driven block
 After you have completed these steps, the new content block becomes a part of your model documentation.
 
 <!--- TO DO We might need to revisit terminology: we talk about "block", "section", and "module", but they all seem to be names for very similar things.--->
+### Add mathematical formulas
```

Comment:
As a general comment, tasks should have numbered steps to show me how to do something. Your text tells me that I can do something but _how do I actually add a formula_? An example or a screenshot with an animated GIF that shows how to do this would also be helpful as this feature is kinda nestled away. 

(The CK Editor docs are lacking here as well, BTW, though they do have a demo of sorts.)